libs/wingsdk/src/std/json.ts

```diff
@@ -42,15 +46,18 @@ export class Json {
    * Formats Json as string
    *
    * (JSON.stringify($args$))
-   *
-   * @macro ((args) => { return JSON.stringify(args[0], null, args[1]) })([$args$])
+
+   * @macro ((args) => { return JSON.stringify(args[0], null, args[1].indent) })([$args$])
    *
    * @param json to format as string
    * @returns string representation of the Json
    */
-  public static stringify(json: Json | MutJson, indent?: number): string {
+  public static stringify(
+    json: Json | MutJson,
+    options: JsonStringifyOptions = { indent: 0 }
```

Comment:
I don't think we can (or should) rely on a default option here

```suggestion
    options?: JsonStringifyOptions
```

Reply:
Yeah, @Chriscbr confirmed that the compiler ignores that.